src/components/search/Search.tsx

```diff
@@ -38,6 +38,9 @@ const Search = () => {
     /* Base variables */
     const searchResults = useAppSelector(getSearchResults);
     const searchSpecimen = useAppSelector(getSearchSpecimen);
+    const pageSize = 25;
```

Comment:
We could also set 25 as default value, currently default is 10

Reply:
Yes, that would be good, this number is specially for the paginator component, so not dependent on the API (but used in the params with the call). I think 25 is a good number, not too big, not too small, otherwise we could increase or decrease it a little